tests/unit/commands/test-test.js

```diff
@@ -50,10 +54,7 @@ describe('test command', function() {
     });
 
     it('builds and runs test', function() {
-      return command.validateAndRun([]).then(function() {
-        td.verify(tasks.Build.prototype.run(), {ignoreExtraArgs: true});
-        td.verify(tasks.Test.prototype.run(), {ignoreExtraArgs: true});
-      });
+      return command.validateAndRun([]);
```

Comment:
What is this testing now?

It used to test that `validateAndRun` would call `run`, adding something like
```js
    if (true) {
      console.log('bye bye');
      return;
    }
```
to the top of `validateAndRun` does not cause this test to fail.